nextjs/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "NODE_OPTIONS='--perf-basic-prof' next start -p 5000",
```

Comment:
Interesting! I was curious why `--perf-basic-prof` wasn't in the Dockerfile. 
Is this a nice way of making sure to _always_ run with the arguments and never forget, even outside of Docker? Pretty cool!

Reply:
No hard reason, just felt it will be easier to know what is running just from the commands.

Reply:
I would prefer this to be more visible. Dockerfile sounds great!

Reply:
Done, changed it.